src/main/java/org/sagebionetworks/bridge/services/OrganizationService.java

```diff
@@ -162,6 +169,12 @@ public void deleteOrganization(String appId, String identifier) {
         
         Organization existing = orgDao.getOrganization(appId, identifier)
                 .orElseThrow(() -> new EntityNotFoundException(Organization.class));        
+        if (assessmentDao.hasAssessmentFromOrg(appId, identifier)) {
+            throw new ConstraintViolationException.Builder().withMessage(
+                    "Cannot delete organization: it is referenced by an assessment that is still accessible")
```

Comment:
How about: "Cannot delete organization (it still owns one ore more assessments)."